elements/pfe-hide-show/src/pfe-hide-show.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2018 Red Hat, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import PFElement from "../pfelement/pfelement.js";
+
+class PfeHideShow extends PFElement {
+  static get tag() {
+    return "pfe-hide-show";
+  }
+
+  get styleUrl() {
+    return "pfe-hide-show.css";
+  }
+
+  get templateUrl() {
+    return "pfe-hide-show.html";
+  }
+
+  get isTab() {
+    return this.parentNode
+      ? this.parentNode.offsetWidth > 768
```

Comment:
@starryeyez024 - I think an improvement here could be to have the ability to pass the min width via an attribute or property? Or maybe not? I guess it depends on how opinionated we want to be.

Reply:
@kylebuch8 way ahead of you good buddy (by 3 days, ha) https://github.com/patternfly/patternfly-elements/issues/275